pkg/pipeline/provider.go

```diff
@@ -41,22 +41,19 @@ func Provider() *schema.Provider {
 			},
 		},
 
-		ResourcesMap: util.AddTelemetry(
-			productId,
-			map[string]*schema.Resource{
-				"pipeline_source":              pipelineSourceResource(),
-				"pipeline_project_integration": pipelineProjectIntegrationResource(),
-				"pipeline_node_pool":           pipelineNodePoolResource(),
-				"pipeline_node":                pipelineNodeResource(),
-			},
-		),
-
-		DataSourcesMap: util.AddTelemetry(
-			productId,
-			map[string]*schema.Resource{
-				"pipeline_project": projectDataSource(),
-			},
-		),
+		ResourcesMap: map[string]*schema.Resource{
```

Comment:
You can't remove telemetry

Reply:
Not sure what you mean by that.

Reply:
In your deleted code there was `DataSourcesMap: util.AddTelemetry(`

You have to keep that. We use that to give us metrics on what's getting used and not